p/i/index.php

```diff
@@ -38,13 +36,16 @@
 			@filemtime(join_path(DATA_PATH, 'config.php'))
 		);
 		if (httpConditional($dateLastModification, 0, 0, false, PHP_COMPRESSION, true)) {
+			Minz_Session::init('FreshRSS');
+			Minz_Session::_param('keepAlive', 1);	//To prevent the PHP session from expiring
 			exit();	//No need to send anything
 		}
 	}
 
 	try {
 		$front_controller = new FreshRSS();
 		$front_controller->init();
```

Comment:
This line also calls `Minz_Session::init('FreshRSS');`

Reply:
That would also be fine. We are calling `new FreshRSS()` from `actualize_script.php` as well (which should be updated or replaced by our CLI approach some day)